### ISSUE TITLE
N4944 pre-CD: Remaining inconsistent inline constexpr variable templates

### ISSUE THREAD

**Issue by Dani-Hub** [cplusplus/draft#6204]
After the recent adjustment to strike "inline" from constexpr variable templates, the following cases are still present (looking at the N4944 pre-CD working draft):

1) [ranges.syn]:
```
namespace views {
  template<size_t N>
    inline constexpr unspecified adjacent = unspecified; // freestanding
  [...]
}
[...]
namespace views {
  template<size_t N>
    inline constexpr unspecified adjacent = unspecified; // freestanding
  [...]
}
[...]
namespace views {
  template<size_t N>
    inline constexpr unspecified adjacent_transform = unspecified; // freestanding
  [...]
}
```
2) [range.refinements]:
```
template<class R>
  inline constexpr bool is-initializer-list = see below ; // exposition only
```

**Comment by jwakely** [comment]
I agree.

**Comment by tkoeppe** [comment]
Thank you both, sounds good! Does either of you fancy making a PR? :-) If
not, I'll get to it in a bit.

**Comment by tkoeppe** [comment]
Thanks a lot!

**Comment by frederick-vs-ja** [comment]
This should have been fixed.

**Comment by AlisdairM** [comment]
This was most recently addressed by #7240.  Does that resolve this issue?
